llvm/include/llvm/SandboxIR/Tracker.h

```diff
@@ -291,6 +291,22 @@ class MoveInstr : public IRChangeBase {
 #endif // NDEBUG
 };
 
+class InsertIntoBB : public IRChangeBase {
```

Comment:
`class InsertIntoBB final : public IRChangeBase {` ?

Reply:
Yeah, I think all of them should be marked as `final`.